spec.html

```diff
@@ -4339,10 +4340,13 @@
           1. If _ny_ is *+&infin;*, return *true*.
           1. If _ny_ is *-&infin;*, return *false*.
           1. If _nx_ is *-&infin;*, return *true*.
-          1. If the mathematical value of _nx_ is less than the mathematical value of _ny_ &mdash;note that these mathematical values are both finite and not both zero&mdash;return *true*. Otherwise, return *false*.
+          1. If the mathematical value of _nx_ is less than the mathematical value of _ny_ &mdash; note that these mathematical values are both finite and not both zero &mdash; return *true*. Otherwise, return *false*.
```

Comment:
The emdash should not have spaces around it (delete the extra space after `_ny_` instead).